packages/cli-core/src/package-manager-controller/package_manager_controller.ts

```diff
@@ -112,6 +120,45 @@ Get started by running \`${this.binaryRunner} amplify sandbox\`.`;
     );
   }
 
+  /**
+   * runWithPackageManager - Factory function that runs a command with the specified package manager's binary runner
+   */
+  runWithPackageManager(
+    args: string[] = [],
+    dir: string,
+    options?: {
+      env?: Record<string, string>;
+      stdin?: 'inherit' | 'pipe' | 'ignore';
+      stdout?: 'inherit' | 'pipe' | 'ignore';
+      stderr?: 'inherit' | 'pipe' | 'ignore';
+      extendEnv?: boolean;
+    }
```

Comment:
Done and reverted [30cd61e](https://github.com/aws-amplify/amplify-backend/pull/947/commits/30cd61ee0bb883b31eb59e104cde07a6038231e7), since it failed the build.

I'm trying again

Reply:
Done.